projects/ngx-datatable/src/lib/utils/math.spec.ts

```diff
@@ -34,4 +34,27 @@ describe('Math function', () => {
       });
     });
   });
+
+  describe('adjustColumnWidths', () => {
+    describe('flex mode', () => {
+      it('should not go overflow/underflow compared to given max width', () => {
+        const cols = {
+          center: [
```

Comment:
Just out of curiosity: why do you additionally wrap the array here, when you always just access `cols.center`?

Otherwise, you could just go with a simple array here, as in all other tests:

```ts
const columns = [
  ...
```

Reply:
I didn't realise using groups is not required, I've simplified the test.